### DISCUSSION TITLE
Units in examples

### DISCUSSION THREAD

**Discussion by oppenjaime** [magpylib/magpylib#745] [Q&A]
Hi, I'm coming back to magpylib since a while and I've noticed outstanding updates. I was checking the "Coils" examples, and I got a bit confused about the units used in the plots. I understand that now everything is measured in SI units, e.g metres for distances/lenghts, Amps for current and Tesla for magnetic field. I understand that coils model 1 & 2 and the Helmholtz tutorial should be in metres, but the plot axis labels are in mm and the magnetic field colorbar is in mT, and apparently there are no scaling factors in the program to explain that change. Thanks for your help.

**Comment by OrtnerMichael** [comment]
hi @oppenjaime,

thank you for pointing this out - the units are put by hand and were incorrect. I fixed the error in  #746. This will be updated in the docu today.

It might also be interesting for you to know:
" .... , for advanced use one should be aware that the analytical solutions are scale invariant - “a magnet with 1 mm sides creates the same field at 1 mm distance as a magnet with 1 m sides at 1 m distance”. The choice of length input unit is therefore not relevant, but it is critical to keep the same length unit for all inputs in one computation."

https://magpylib.readthedocs.io/en/latest/_pages/docu/docu_magpylib_api.html#docu-magpylib

Please close the discussion if you are satisfied with the reply. I'd be happy to answer more :)

br michael

**Reply by OrtnerMichael** [reply]
the docu is fixed now.

the docu default version now points at "latest", not at "stable", so you have to access the page again, not  just reload.

br michael

**Reply by oppenjaime** [reply]
Thanks a lot Michael, yesterday I realised that I could have verified against the simple calculation of the loop formula to verify the units, which later I did and confirmed that only were the units in the chart.
